src/main/resources/database/changes/release-29/case_action_addition.sql

```diff
@@ -0,0 +1,63 @@
+-- Adding required indexes for case_action view.
+Create index idx_casegroup_collection_exercise_id
```

Comment:
A little picky, but can we have SQL keywords in upper case, as thats the convention elsewhere in this project.

i.e.
```
CREATE INDEX idx_casegroup_collection_exercise_id
```

Reply:
👍